nixos/modules/services/web-servers/unit/default.nix

```diff
@@ -0,0 +1,70 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.unit;
+
+  configFile = builtins.toFile "unit.json" ''
+    ${cfg.config}
+  '';
+
+in {
+  options = {
+    services.unit = {
+      enable = mkEnableOption "Unit App Server";
+      package = mkOption {
+        type = types.package;
+        default = pkgs.unit;
+        description = "Unit package to use.";
+      };
+      stateDir = mkOption {
+        default = "/var/spool/unit";
+        description = "Unit data directory.";
```

Comment:
Does this need to be configurable? I don't think there's any need for it. Same for `logDir` tbh.

Reply:
It is possible to leave these parameters?

Reply:
Do you need them? Why would anybody need them?

Reply:
Everything connected to the web-apps, I drag in a folder /var/data/web and allow access to it only to a specific user.